### DISCUSSION TITLE
Where is search.json generated

### DISCUSSION THREAD

**Discussion by curtisduggan** [timlrx/tailwind-nextjs-starter-blog#720] [Q&A]
kbar uses search.json to populate the universe of searchable things.

How do I add to that — let's say I created a custom page with a directory of links to restaurants with cards for each restaruant (for instance).

How would I make it so you can command-k search the restaurants?

**Comment by timlrx** [comment]
@curtisduggan see if this helps? https://github.com/timlrx/tailwind-nextjs-starter-blog#how-can-i-customize-the-kbar-search
